extern-api-java-sdk-integrational-tests/src/test/java/ru/kontur/extern_api/sdk/DocflowServiceIT.java

```diff
@@ -670,7 +674,8 @@ void testPrintFromDocflows(QueryContext<Docflow> docflowCxt) {
         }
     }
 
-    private static List<QueryContext<Docflow>> createDocflows(ExternEngine engine, TestData[] testData) {
+    private static List<QueryContext<Docflow>> createDocflows(ExternEngine engine,
+            TestData[] testData) {
```

Comment:
выравнивание аргуменов -- друг под другом
```suggestion
    private static List<QueryContext<Docflow>> createDocflows(
            ExternEngine engine,
            TestData[] testData
    ) {
```